spec.html

```diff
@@ -12612,20 +12615,20 @@ <h1>
         </h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>It returns a Boolean value which is *true* if and only if _Desc_ can be applied as the property of an object with specified _extensibility_ and current property _current_ while upholding <emu-xref href="#sec-invariants-of-the-essential-internal-methods">invariants</emu-xref>. When such application is possible and _O_ is not *undefined*, it is performed for the property named _P_ (which is created if necessary).</dd>
+          <dd>It returns a Boolean value which is *true* if and only if _Desc_ can be applied as the property of an object with specified _extensibility_ and current property _current_ while upholding <emu-xref href="#sec-invariants-of-the-essential-internal-methods">invariants</emu-xref>. When such application is possible and _O_ is not *undefined*, it is performed for the property named _P_ (which is created if necessary). For the purposes of this abstract operation, when we say an attribute is "set by _Desc_ or default", it means that if _Desc_ has a field of the same name as the attribute, the attribute is set to the value of that field; otherwise the attribute is set to its <emu-xref href="#table-object-property-attributes">default value</emu-xref>.</dd>
```

Comment:
In the editor call today, we discussed this phrasing, and we do not think it adds clarity. Can you revert this and its usages? We think the original phrasing was clear enough.

Reply:
I eliminated the "set by _Desc_ or default" phrase, replacing it with a consistent blend of the two phrasings in the status quo, plus @ljharb's "if it has the field" suggestion. See 0cd53f1.